src/utils/html-to-portable-text.ts

```
@@ -0,0 +1,481 @@
+      const element = node as Element
+      const tagName = element.tagName.toLowerCase()
+
+      switch (tagName) {
+        case 'p':
+          const block: Block = {
+            _type: 'block',
+            _key: nanoid(),
+            style: 'normal',
+            children: [],
+            markDefs: [],
+          }
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)